### DISCUSSION TITLE
Support for custom log4j xml?

### DISCUSSION THREAD

**Discussion by 0x-2a** [itzg/docker-minecraft-server#3192] [Q&A]
Other than forking the repo and rolling my own container, is there a way to make my own custom `log4j2.xml` and reuse it across containers? For example, I like to config log4j to output to different log files by log level.

**Comment by itzg** [comment]
Correct, at this time it can't be given an alternate log config file; however, that would be a pretty simple enhancement like

https://github.com/itzg/docker-minecraft-server/blob/0582a8ab97d68bca297bca555ebb46c4337afeb4/scripts/start-finalExec#L106

If want to PR that change, that would be great.
